.github/workflows/publish-cli-bins.yaml

```
@@ -9,74 +9,146 @@ on:
+          # Verify binary integrity
+          if ! [ -x $NAME ]; then
+            echo "Error: Binary is not executable"
+            exit 1
+          fi
+
+          # Verify version output format
+          if ! $NAME version; then
+            echo "Error: Version command failed"
+            exit 1
+          fi
+ 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)